package.json

```diff
@@ -14,18 +14,20 @@
     "chalk": "^4.1.0",
     "future-event": "^1.5.0",
     "glob": "^7.1.6",
-    "google-closure-library": "^20200719.0.0",
+    "google-closure-library": "^20201102.0.1",
     "jest": "^26.1.0",
     "jsdom": "^16.3.0"
   },
   "scripts": {
-    "build": "clean && mvn -DskipTests",
-    "build:fast": "mvn -DskipTests -pl externs/pom.xml,pom-main.xml,pom-main-shaded.xml",
-    "clean": "yarn runtime_tests:clean && mvn clean",
-    "runtime_tests:build": "$npm_package_config_runtime_tests_dir/utils/build.sh",
-    "runtime_tests:clean": "rm -rf $npm_package_config_runtime_tests_dir/**/build/*",
-    "runtime_tests:run": "jest --testRegex $npm_package_config_runtime_tests_dir/utils/test/* --verbose=true",
-    "runtime_tests": "yarn runtime_tests:clean && yarn run --silent runtime_tests:build && yarn run --silent runtime_tests:run",
-    "test": "mvn test"
+    "build": "bazel build //:compiler_unshaded_deploy.jar",
+    "build:all": "bazel build :all",
+    "compile": "java -jar bazel-bin/compiler_unshaded_deploy.jar",
+    "clean": "yarn runtime_tests:clean && bazel clean",
+    "runtime_tests:clean": "rm -rf $npm_package_config_runtime_tests/**/build/*",
+    "runtime_tests:build": "$npm_package_config_runtime_tests/utils/build.sh",
+    "runtime_tests:run": "jest --testRegex $npm_package_config_runtime_tests/utils/test/* --verbose=true",
+    "runtime_tests": "yarn --silent runtime_tests:clean && yarn --silent runtime_tests:build && yarn run runtime_tests:run",
+    "test": "bazel test //:compiler_unshaded_deploy.jar",
```

Comment:
There's nothing to test here. Bazel makes a distinction between targets that are tests and targets that are not.

Reply:
Can you leave a more helpful suggestion, like how it should be changed?

Reply:
Oh, can we just delete it? It won't do anything. I think "test:all" covers it, if it's even worth making a distinction.

Reply:
I can absolutely delete it, I just have the script here so people can interface completely with `yarn`/`npm` without needing to deal with Bazel-y stuff: `yarn build`, `yarn test`, etc.

I will actually probably add the `runtime_tests` test to `yarn test`, such that `yarn test` is: ```bazel test:all && runtime_tests```

Thoughts? It's just a way to do all the Bazel logic for OSS people ahead of time so they can just easily interface with the package.json scripts.

Reply:
I would probably make `test` = `bazel test //:all` and `test:all` = `bazel test //:all && runtime_tests`, but whatever you decide is fine. The important thing is that `bazel test //:compiler_unshaded_deploy.jar` is a no-op.

Reply:
Awesome, I will make those changes right now.